types/postcss-prefix-selector/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for postcss-prefix-selector 1.14
```

Comment:
```suggestion
// Type definitions for postcss-prefix-selector 1.15
```
https://github.com/RadValentin/postcss-prefix-selector/releases/tag/v1.15.0
(released)